R/metadata.R

```diff
@@ -454,7 +454,8 @@ prevent_multiple_calls <- function(packet, name, call) {
   if (!is.null(packet$orderly2[[name]])) {
     cli::cli_abort(
       c("Only one call to 'orderly2::orderly_{name}' is allowed",
-        i = "You have already called this function earlier in your orderly.R"),
+        i = paste("You have already called this function earlier",
+                  "in your <reportname>.R")),
```

Comment:
Pass the filename down here to `prevent_multiple_calls` and use it in the error message, it'll be clearer.

Reply:
done, just did it using the packet info we have, thought that was simpler